src/components/reporter/widgets/Widgets.module.scss

```diff
@@ -43,6 +43,30 @@
 }
 
 .Textarea {
-  height: 150px;
+  height: 250px;
   font-family: inherit;
+  resize: none;
```

Comment:
We'll probably want to keep the resize since some wardens use this window to compose the message in, and we want to give them as much room as they need. It does break the layout, but in a way I accept as necessary for now 😄

Reply:
Ok!